.changeset/spotty-paws-think.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/cli': patch
+'@backstage/create-app': patch
+---
+
+Support .npmrc when building with private NPM registries
```

Comment:
```suggestion
Support `.npmrc` when building with private NPM registries
```

Reply:
Good call! Didn't occur to me since I filled that in the CLI. Let's see

Reply:
Fixed, thanks!